app/views/admin/organizations/invoices/edit.html.haml

```diff
@@ -1,53 +1,61 @@
-%header.with-subtitle.padded
-  %h1
-    Edit #{@invoice.display_name} for #{link_to @invoice.organization.short_name, admin_organization_invoices_path(@invoice.organization)}
+%h1
+  Edit #{@invoice.display_name} for #{link_to @invoice.organization.short_name, admin_organization_invoices_path(@invoice.organization)}
 
-.well{ style: "margin-top: 20px;" }
-  %p
-    Active:
-    %strong
-      - if @invoice.active?
-        "Yes"
-      - elsif @invoice.was_active?
-        Expired
-      - else
-        no
-  %p
-    - if @invoice.expired?
-      Expired:
-    - else
-      Expires:
-    - if @invoice.subscription_end_at.present?
-      %span.convertTime
-        = l @invoice.subscription_end_at, format: :convert_time
-  %p
-    Associated invoices:
-    %ul
-      %li
-        Previous:
-        - if @invoice.previous_invoice.present?
-          #{link_to @invoice.previous_invoice.display_name, edit_admin_organization_invoice_path(organization_id: @invoice.organization.to_param, id: @invoice.previous_invoice.id)}
-      %li
-        Next:
-        - if @invoice.following_invoice.present?
-          #{link_to @invoice.following_invoice.display_name, edit_admin_organization_invoice_path(organization_id: @invoice.organization.to_param, id: @invoice.following_invoice.id)}
+%table.table-list
```

Comment:
Does this look really spread out on a big screen? Would it be better in multiple columns?